tests/integration/components/observation-form/component-test.js

```diff
@@ -1,26 +1,90 @@
+  });
+
+  test('inputs renders', async function (assert) {
+    assert.dom('[data-test-input-pikaday]').exists();
+    assert.dom('[data-test-input-birdname]').exists();
+    assert.dom('[data-test-input-location]').exists();
+    assert.dom('[data-test-input-notes]').exists();
+    assert.dom('[data-test-button-cancel]').exists();
+    assert.dom('[data-test-button-clear]').exists();
+    assert.dom('[data-test-button-save]').exists();
+  });
 
-    assert.dom(this.element).hasText('');
+  test('pikaday input', async function (assert) {
+    await click('[data-test-input-pikaday]');
+    await Pikaday.selectDate(new Date(2022, 8, 28));
+    assert.strictEqual(Pikaday.selectedYear(), '2022');
+    assert.strictEqual(Pikaday.selectedMonth(), '8');
+    assert.strictEqual(Pikaday.selectedDay(), '28');
+    await closePikaday('[data-test-input-pikaday]');
+  });
 
-    // Template block usage:
-    await render(hbs`
-      <ObservationForm>
-        template block text
-      </ObservationForm>
-    `);
+  test('location input disabled', async function (assert) {
+    assert.dom('[data-test-input-location]').hasNoAttribute('disabled');
+  });
+
+  test('location input change to disabled', async function (assert) {
+    await click('[data-test-input-location]');
+    assert.dom('[data-test-input-location]').hasAttribute('disabled');
+  });
+
+  test('fill in the form', async function (assert) {
+    assert.dom('[data-test-button-save]').hasAttribute('disabled');
+
+    const stringDate = moment(this.observation.observationDate).format(
+      'YYYY-MM-DD'
+    );
+    await click('[data-test-input-pikaday]');
+    await Pikaday.selectDate(new Date(stringDate));
+
+    await closePikaday('[data-test-input-pikaday]');
+
+    await fillIn('[data-test-input-birdname]', `${this.observation.birdname}`);
+    await fillIn('[data-test-input-notes]', `${this.observation.notes}`);
+    await fillIn(
+      '[data-test-input-location]',
+      `latitude: ${this.observation.latLocation.toFixed(
+        6
+      )} / longitude: ${this.observation.lngLocation.toFixed(6)}`
+    );
+
+    assert.dom('[data-test-input-pikaday]').hasValue(stringDate);
+    assert
+      .dom('[data-test-input-birdname]')
+      .hasValue(`${this.observation.birdname}`);
+    assert
+      .dom('[data-test-input-location]')
+      .hasValue(
+        `latitude: ${this.observation.latLocation.toFixed(
+          6
+        )} / longitude: ${this.observation.lngLocation.toFixed(6)}`
+      );
+    assert.dom('[data-test-input-notes]').hasValue(`${this.observation.notes}`);
+  });
 
-    assert.dom(this.element).hasText('template block text');
+  test('clear the form', async function (assert) {
+    await click('[data-test-button-clear]');
+    assert.dom('[data-test-input-pikaday]').hasValue('');
+    assert.dom('[data-test-input-birdname]').hasValue('');
+    assert.dom('[data-test-input-location]').hasValue('');
+    assert.dom('[data-test-input-notes]').hasValue('');
```

Comment:
tak możesz sprawdzić czy nie ma value, powinno zadziałać

```suggestion
    assert.dom('[data-test-input-pikaday]').hasNoValue()
    assert.dom('[data-test-input-birdname]').hasNoValue()
    assert.dom('[data-test-input-location]').hasNoValue()
    assert.dom('[data-test-input-notes]').hasNoValue()
```